### ISSUE TITLE
Last Minute Request :)

### ISSUE THREAD

**Issue by Shan009** [Mindgreppers/HimachalCodeCamp2016#19]
Hi, 
Sorry I am late sending my application for the camp , I guess i was out of luck , just stumbled upon your website today, I am hoping if you can get one more application or , if some one cancelled and you have a vacant place, please let me know.

I would love to jump in 
Thanks and Regards
Shan

**Comment by mastersilv3r** [comment]
Hi Shan009

Can you let us know a bit about yourself. How much experience in programming? What else about you should we know.

@jaagameister will take a call on this.

**Comment by Shan009** [comment]
Hi, 
Thanks for taking it in consideration.
I don't have much of an experience in programming that's why i wanna join in, but i have been a freelancer for almost 4 years now, have experiences in handling lot of different types of projects and Designs , as i work alone so i haven't got much into the programming side of the story. 
Also Code Camp interested me with all the experiences i might gain. 
Frankly never been to camps like this and i am only 21 so i am still learning and really would love to join in.
I didn't know if there were any criteria of experience necessary, so i thought i would join as learner , 

Thanks 
Shan

P.S - Let me know if any specifics you want to know about me.

**Comment by mastersilv3r** [comment]
Thanks for sharing Shan. So your experience is primarily in design is it?

**Comment by Shan009** [comment]
Design and Website Developments , basically i have got expertise in all kind of CMS , like Wordpress, Joomla , Drupal and Opencart 
In Design i have worked on banners , flyers , wireframes , illustrating 
Basically i make websites , with some knowledge of programming and php

**Comment by mastersilv3r** [comment]
We are including design sprint as part of our project sprints. Your
presence can be valuable through guidance/contribution in design to an open
source project, while you can code in parallel too.

Are you willing to commit to full six week term?

**Comment by Shan009** [comment]
Yes actually i am , that what interested me the most , I loved the idea of coding in the hills and camps , I love the nature. :)

**Comment by mastersilv3r** [comment]
Alright Shan. You seem to be a good fit. Come on over :)

**Comment by Shan009** [comment]
How? I mean whats the procedure ??

**Comment by mastersilv3r** [comment]
Please fill up the form online (the link is on wiki), add there that you
are confirmed by Ayush and make a bank transfer of Rs. 3000 as advance to
Anuj

Name: Anuj Kumar
HDFC Bank A/C No.: 50100120389983
IFSC: HDFC0000605
Branch: Dharmshala

Once done, send an email to me, freeman@jaagai.in and anuj@gai.co.in
telling you have made advance transfer and please also attach confirmation
details in the mail.

4th is closeby. @jaagameister may add you to the mailing list for the camp
(if he gets time/checks this mail. He is super busy with wrapping up the
jaaga bangalore setup). Also suggest you follow this project on github for
updates on happenings and things to do.

**Comment by Shan009** [comment]
Ok will do it tomorrow and update you..

**Comment by Shan009** [comment]
and i have already filled in the wiki before opening the issue

**Comment by Shan009** [comment]
Hi, 
I need a mobile number associated to the account to do the transfer as i am using Internet Banking Mobile app.

**Comment by mastersilv3r** [comment]
Please send an email to anuj and ask.

**Comment by mastersilv3r** [comment]
Anuj.k@gai.co.in

**Comment by Shan009** [comment]
So I have to reach there on 4th only , or I can be late , I am still figuring out a way to reach there , Also can you guide me with packing and what to take , cuz i have never been to that part of the country and pretty rookie at this .

**Comment by jinayG** [comment]
Hi Ayush, 
I have the same scenario as shan, I had emailed at study@jaaga.in to join on 4th but did not receive a reply. I will post the same here. Please reply here

Hi, 
I have applied for the coding camp to happen on 4th of April on 22nd March.

It is my dream to participate in camp like this where there is coding, yoga and nature and I understand that I might be late but I humbly request you at accept my application for this camp. With an engineering background and a startup founding on my hand I have a lot of experience that I can contribute in this camp and bring about a positive atmosphere while adhering to the discipline of the camp.

I am about to start on my second startup, working on solar products and would love to gain this experience and develop the passion for coding from the jaaga folks.

I would eagerly wait for your reply to click on the book button on IRCTC website.

Thanks, 
Jinay Gala
EazyTrace Solutions, 
D-27, Sopariwala Estate,
Opera House, Mumbai - 400004
www.eazytrace.com

**Comment by jinayG** [comment]
Also the email ids mentioned by Ayush might be correct as my mails to those ids are failing to get delivered.

**Comment by mastersilv3r** [comment]
4th is best. Where are you coming from? Come via Chandigarh or Delhi, then
bus or train to pathankot then bus, if you are coming from far.

Read about the travel tips on the wiki/travel

Can you please raise an issue for what all to bring? Even other codecampers
need to know. I shall follow up there today.

**Comment by mastersilv3r** [comment]
On Wed, Mar 30, 2016 at 10:25 AM, jinayG notifications@github.com wrote:

> Jinay Gala

Did you fill the application online?

**Comment by jinayG** [comment]
Yeah, I have filled it.

**Comment by Shan009** [comment]
I have made the Transfer to Anuj's Account , 
I will be coming from Kolkata , thinking of boarding a flight to Delhi and then Bus to Dharamshala , What you think ?

**Comment by mastersilv3r** [comment]
Try chandigarh too. Maybe its cheaper?

**Comment by Shan009** [comment]
there are no direct flights to chandigarh and nor to Gaggal Airport , will have to take connecting flights form Delhi which is way to expensive in regard to other means of transport.

**Comment by mastersilv3r** [comment]
Jinay, I like you and I think you can add value to the camp but am afraid
we are loaded with participants at the moment. Freeman is busy with
wrapping up their camp at Jaaga. Let us wait for a day or so for him to
respond?

Sorry buddy but I hope you will understand that the ship shouldn't be
overload too! Can't keep going on making exceptions.

**Comment by mastersilv3r** [comment]
Sounds good.

**Comment by Shan009** [comment]
I have raised another issue for what to bring. 
btw  transfer is made , so do i send email with the details to Anuj and Freeman , and do i fill the form again ??

**Comment by mastersilv3r** [comment]
Please send transaction details to Anuj. That should be good.

**Comment by Shan009** [comment]
Ok details sent.  Anything else you guys need from my part let me know

**Comment by jinayG** [comment]
Hi Ayush,
Thanks for considering it. I just hope I make the cut. Am eager about this
than anything else. Hope Freeman considers it as well.

**Comment by Shan009** [comment]
Hi Ayush 
I just wanted to know that do i go ahead and do my flight bookings already or do i wait for the confirmation mail.
Thanks

**Comment by mastersilv3r** [comment]
Go ahead my friend. You are confirmed already.

**Comment by Shan009** [comment]
Thanks , looking forward to meeting you guys :)

**Comment by Shan009** [comment]
Also please guide me with that other issue of what to pack.

**Comment by Shan009** [comment]
Hey Ayush , I wanted to ask , is there any guy attending from my part of the country or some where near, it would have been great to get acquainted and travel together to reach there.

**Comment by jinayG** [comment]
Hi Ayush,
I am still hoping to hear a positive reply from freeman. Could please
contact him and check the feasibility?
Jinay, I like you and I think you can add value to the camp but am afraid
we are loaded with participants at the moment. Freeman is busy with
wrapping up their camp at Jaaga. Let us wait for a day or so for him to
respond?

Sorry buddy but I hope you will understand that the ship shouldn't be
overload too! Can't keep going on making exceptions.

**Comment by Shan009** [comment]
Hey , Ayush Can i get your cell number please?
